cluster/network/definition.yaml

```diff
@@ -33,13 +33,13 @@ metadata:
           - required: true
             customError: Cluster ref is required.
 spec:
+  group: aws.platformref.crossplane.io
+  names:
+    kind: XNetwork
+    plural: xnetworks
   claimNames:
```

Comment:
we should probably remove this claim/offering from this network XR - there isn't a use case for users creating namespaced instances of Networks - Networks will instead be created as part of other XRs